xbmc/addons/GUIDialogAddonInfo.cpp

```diff
@@ -444,7 +444,9 @@ void CGUIDialogAddonInfo::OnUninstall()
   if (!CGUIDialogYesNo::ShowAndGetInput(CVariant{24037}, CVariant{750}))
     return;
 
-  CJobManager::GetInstance().AddJob(new CAddonUnInstallJob(m_localAddon),
+  bool removeData = CGUIDialogYesNo::ShowAndGetInput(CVariant{24037}, CVariant{39014});
```

Comment:
From my side should good to have a check and show only if settings or somethings else are available on profile path.

It can confuse a user if he say no and then want to look to a file who was never present.